packages/client/src/classes/client.js

```diff
@@ -30,13 +30,14 @@ class Client {
     //Default headers
     this.defaultHeaders = {
       'Accept': 'application/json',
+      'Content-Type': 'application/json',
       'User-agent': 'sendgrid/' + pkg.version + ';nodejs',
     };
 
     //Empty default request
     this.defaultRequest = {
-      json: true,
-      baseUrl: 'https://api.sendgrid.com/',
+      data: {},
+      baseURL: 'https://api.sendgrid.com/',
```

Comment:
Prefer to make this `baseUrl` in the default request and then adapt it later for axios.

Reply:
Where is an appropriate "later"? Also, if this dict is meant to be merged with the renamed opts in the `request` function, then the `defaultRequest` should have keys that align with what's adapted for axios

Reply:
@childish-sambino can I get some feedback?

Reply:
I think it should be adapted after L105 where the data is merged. We define the structure of default request, merge it with the options provided in the request, and then adapt it to be whatever the underlying client is expecting.